chain/src/chain.rs

```diff
@@ -385,6 +385,10 @@ impl ChainService {
                     error!("notify new_uncle error {}", e);
                 }
             }
+            let block_ref: &BlockView = &block;
+            self.shared
+                .notify_controller()
+                .notify_new_block(block_ref.clone());
```

Comment:
```suggestion
                .notify_new_block(block);
```

Reply:
I prefer breaking change in the configuration file than maintaining multiple conf version compatibility.